cloud.profiles.d/edx.yml

```diff
@@ -0,0 +1,21 @@
+edx:
+  provider: mitx
+  size: m3.xlarge
```

Comment:
We're using xlarge currently in production, but we can try having lighter instances like this.

Note that this is the current way that the number of gunicorn workers is calculated: https://github.com/edx/configuration/blob/master/playbooks/roles/edxapp/templates/lms_gunicorn.py.j2#L20
Currently in production, `worker_core_mult.lms` is set to 4.

My biggest concern would be the celery workers (defined in this file), which need to be scaled back for this - we've had to scale them back a couple of times in the past due to memory constraints on our current instances.